### ISSUE TITLE
Graceful handling of semicolon missing at end of return statement

### ISSUE THREAD

**Issue by mpaluchowski** [cflint/CFLint#251]
This code:

```coldfusion
component {
  function foo() {
    return bar
  }
}
```

with the semicolon missing at the end of the `return` statement will currently cause the following error, stopping analysis of the whole file:

```xml
<issue severity="ERROR" id="PARSE_ERROR" message="PARSE_ERROR" category="CFLint" abbrev="PE">
  <location file="foo.cfc" fileName="foo.cfc" function="" column="2" line="4" message="Unable to parse" variable="">
    <Expression><![CDATA[no viable alternative at input 'returnbar}']]></Expression>
  </location>
</issue>
```

Can we have this handled gracefully? So that the analysis reports a `MISSING_SEMI` issue and continues with the rest of the file?

**Comment by ryaneberly** [comment]
Some missing semis are now handled gracefully, including this one.
